packages/app-store/amie/config.json

```diff
@@ -12,5 +12,6 @@
   "email": "support@cal.com",
   "description": "The joyful productivity app\r\r",
   "__createdUsingCli": true,
-  "dependencies": ["google-calendar"]
+  "dependencies": ["google-calendar"],
+  "dirName": "amie"
```

Comment:
I am not sure if you are working on it, but `dirName` should be removed from everywhere.